docs/includes/framework-compatibility-laravel.rst

```diff
@@ -0,0 +1,16 @@
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - {+odm-short+} Version
+     - Laravel 10.x
+     - Laravel 9.x
```

Comment:
Q: Is "9.x" here just for completeness? Or do we expect to use it in the future?

Reply:
I included it for completeness since it's currently a [supported version of Laravel](https://laravelversions.com/en), so users might wonder whether it is compatible.

Reply:
You may add the first line of the table then. 


9.x | 3.9.x | ❌

Reply:
Then this doesn't tell which version is supported.

Reply:
I interpreted this as 3.9.x is incompatible with 10.x, but is compatible with 9.x and added that row to the [new table](https://preview-mongodbcchomongodb.gatsbyjs.io/laravel/DOCSP-35932-compatibility/compatibility/). Is that incorrect? Would you like me to keep/remove/change anything before merging?

Reply:
Sorry, I mean 3.9 is no longer maintained.

Reply:
Lets remove this last line then. We already have a link to the old versions table.